snapcraft/_store.py

```diff
@@ -395,14 +402,14 @@ def update_metadata(snap_yaml, force):
     # hit the server
     store = storeapi.StoreClient()
     with _requires_login():
-        store.update_metadata(snap_name, metadata, force)
+        store.push_precheck(snap_name)
+        store.push_metadata(snap_name, metadata, force)
 
-    logger.info("The metadata has been updated")
+    logger.info("The metadata has been pushed")
```

Comment:
Just one minor nit, can you remove this from here and put it under
```
snapcraft.push_metadata(snap_file, force)
```
in `snapcraft.cli.store`?